readme.md

```diff
@@ -94,40 +134,109 @@ Alternatively, put the metadata in the [`author` or `contributors` fields](https
 
 ## Supported Properties
 
--   `name`: overrides the name stored in git commits
--   `github`: GitHub username
--   `twitter`: twitter username
--   `mastodon`: mastodon with format `@user@domain`
+*   `name`: overrides the name stored in git commits
+*   `github`: GitHub username
+*   `twitter`: Twitter username
+*   `mastodon`: Mastodon with format `@user@domain`
 
 ## API
 
 ### `contributors([options])`
 
 The options object may contain the following properties:
 
--   `limit`: number. Only render the top `<limit>` contributors, sorted by commit count. By default, all contributors are included.
--   `contributors`: array or module id, see above.
--   `cwd`: working directory from which to resolve `contributors` module (if any). Defaults to [`cwd`](https://github.com/vfile/vfile#vfilecwd) of the markdown file, falling back to `process.cwd()` if it doesn't resolve.
--   `appendIfMissing`: boolean. Inject Contributors section if there is none. Default is `false`.
-
-## Install
-
-With [npm](https://npmjs.org) do:
-
-    npm install remark-git-contributors
+*   `limit`: number.
+    Only render the top `<limit>` contributors, sorted by commit count.
+    By default, all contributors are included.
+*   `contributors`: array or module id.
+    See above.
+*   `cwd`: working directory from which to resolve `contributors` module (if
+    any).
+    Defaults to [`cwd`][cwd] of the file, falling back to `process.cwd()` if
+    it doesn’t resolve.
+*   `appendIfMissing`: boolean.
+    Inject Contributors section if there is none.
+    Default is `false`.
 
 ## Contribute
 
-See [`contributing.md` in `remarkjs/remark`](https://github.com/remarkjs/remark/blob/master/contributing.md) for ways to get started.
+See [`contributing.md` in `remarkjs/remark`][contributing] for ways to get
+started.
 
-This organisation has a [Code of Conduct](https://github.com/remarkjs/remark/blob/master/code-of-conduct.md).  By interacting with this repository, organisation, or community you agree to abide by its terms.
+This organisation has a [Code of Conduct][coc].  By interacting with this
+repository, organisation, or community you agree to abide by its terms.
 
 ## Contributors
 
 | Name                | GitHub                                       | Social                                                |
 | :------------------ | :------------------------------------------- | :---------------------------------------------------- |
 | **Vincent Weevers** | [**@vweevers**](https://github.com/vweevers) | [**@vweevers@twitter**](https://twitter.com/vweevers) |
+| **Titus Wormer**    | [**@wooorm**](https://github.com/wooorm)     | [**@wooorm@twitter**](https://twitter.com/wooorm)     |
 
 ## License
 
-[MIT](LICENSE.md) © 2018-present Vincent Weevers
+[MIT][license] © [Vincent Weevers][author]
```

Comment:
I don't intend to update my website so please remove the link (or change it to my github profile, that's fine).

Reply:
Interesting, sure!